<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopTheme</class>
 <widget class="QWidget" name="DesktopTheme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>460</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Theme</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="4">
        <widget class="QPushButton" name="m_fileInstallButton">
         <property name="text">
          <string>Insta&amp;ll from File</string>
         </property>
         <property name="icon">
          <iconset theme="document-open"/>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="5">
        <widget class="QListView" name="m_theme"/>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="m_newThemeButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Get New Themes...</string>
         </property>
         <property name="icon">
          <iconset theme="document-open-remote"/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="m_statusLabel">
         <property name="text">
          <string/>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>462</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Details</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="DesktopThemeDetails" name="m_detailsWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DesktopThemeDetails</class>
   <extends>QWidget</extends>
   <header>desktopthemedetails.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
